CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributors
+
+# Reporting issues
+* Describe (in detail) what should have happened. Include any supporting information (stack traces, errors, etc)
+* Include any steps to replicate the issue
+* Indicate OS and Python versions
+
+# Development environment setup
+1. Clone repository:
+```
+git clone https://github.com/mitre/caldera.git --recursive
+```
+
+2. Create a virtualenv:
+```
+python3 -m venv venv
+. venv/bin/activate
+```
+
+3. Install dependencies:
+```
+pip install -r requirements.txt -r requirements-dev.txt
+```
+
+4. Install the [pre-commit framework](https://pre-commit.com/#install)
```

Comment:
this appears in the requirements-dev as a pip option

Reply:
Updated to remove step